crates/router/src/db/kafka_store.rs

```diff
@@ -1491,21 +1493,50 @@ impl PayoutAttemptInterface for KafkaStore {
         &self,
         this: &storage::PayoutAttempt,
         payout_attempt_update: storage::PayoutAttemptUpdate,
+        payouts: Option<&storage::Payouts>,
         storage_scheme: MerchantStorageScheme,
     ) -> CustomResult<storage::PayoutAttempt, errors::DataStorageError> {
-        self.diesel_store
-            .update_payout_attempt(this, payout_attempt_update, storage_scheme)
+        let updated_payout_attempt = self
+            .diesel_store
+            .update_payout_attempt(this, payout_attempt_update, payouts, storage_scheme)
+            .await?;
+        if let Some(payout) = payouts {
+            let _ = self
```

Comment:
Why are we using `let _`? If it is returning a `Result`, can we use an explicit `match` and log the error, instead of using `let _` to address warnings about the `Result` being unused?

Please make the change elsewhere too.